www/js/model.js

```diff
@@ -191,11 +191,12 @@ define([
 				// Summary: returns a list of all distinct entities author has entered for the model
 				var entityList = new Array();
 				array.forEach(this.authored.getNodes(), function(node){
-					if(node.entity && array.indexOf(entityList, node.entity) == -1){
-						entityList.push(node.entity);
+					//out of the list of entities, we pick the first one
```

Comment:
@riteshsamala I think the subtask was confusing on this one.  The entity dropdown must include ALL the entities the author entered in the entire model, with duplicates removed--not the first entity for each node.  So if the author has three nodes with schema and entities lists of <DRT: Bob;Bob's Plane;Plane> <Tailwind: Bob;Bob's Plane;Plane> <DRT: Jim;Jim's Car;Car> then the list of entities given to the student would be [Bob, Bob's Plane, Plane, Jim, Jim's Car, Car].